src/rnp/rnpcfg.cpp

```diff
@@ -373,6 +373,19 @@ rnp_cfg_getlist(rnp_cfg_t *cfg, const char *key)
     return NULL;
 }
 
+std::string rnp_cfg_getlist_string(const rnp_cfg_t *cfg, const std::string &key, size_t index) {
+    list *lval = rnp_cfg_getlist(const_cast<rnp_cfg_t*>(cfg), key.c_str());
+    if (lval) {
```

Comment:
Here would be better to change to `if (!lval) {return ""}`, to have less nested code.